src/dungeonmastersvault.ts

```diff
@@ -0,0 +1,388 @@
+/** @format */
+
+import createLogger from './log';
+import { getStorage, migrateStorage } from './storage';
+import { IRoll, ThreeDDiceRollEvent, ThreeDDice, ITheme, ThreeDDiceAPI } from 'dddice-js';
+
+import './index.css';
+import './dungeonmastersvault.css';
+import imageLogo from 'url:./assets/dddice-32x32.png';
+
```

Comment:
```suggestion
import notify from './utils/notify';
```

You need to import my, now confusingly named, notify lib for the error messages

Reply:
Done! And yeah, it's unfortunate, uh? 😔 I'd love to get rid of the dependency, but my chat messages will look like a five year old designed them. 😂

Reply:
Maybe we can tackle that after the refactor :D